Makefile

```
@@ -478,6 +478,10 @@ build-archive: | $(RELEASE_DIR)
 	echo $(GITTAG) > teleport/VERSION
 	tar $(TAR_FLAGS) -c teleport | gzip -n > $(RELEASE).tar.gz
 	cp $(RELEASE).tar.gz $(RELEASE_DIR)
+	# Make a centos7 archive when building linux-amd64. The build is identical.
```

Not sure what you're saying here?

```suggestion
	# Make a centos7 archive when building linux-amd64. The actual binaries for centos7 are the same as for other linux-amd64, this is just an extra packaging format.
```

That's probably too verbose, but that what I think is happening here.

---

I'll try rewording. It is actually the same build artifacts with a differently named tarball. There is no longer a separate centos7 archive - the default linux-amd64 one is centos7 compatible.

---

I reworded to "linux-amd64 generates a centos7-compatible archive. Make a copy with the -centos7 label for the releases page. We should probably drop that at some point."